src/components/MerchantsPage/gam/MegaGamListItem.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import { Campaign } from '../../../utilities/api/types';
+import { tabletScreens } from '../../../utilities/general/responsive';
+
+interface Props {
+  campaign: Campaign;
+}
+
+const MegaGamListItem = ({ campaign }: Props) => {
+  const { t } = useTranslation();
+  return (
+    <Container>
+      <HeroImage heroImageUrl={campaign.gallery_image_urls[0]} />
+      <Content>
+        {/* TODO: Figure out if this is the correct header. If so, we need to translate. */}
+        <Header>Mega-GAM</Header>
+        {/* TODO: Don't hard code this. Figure out where we should get this data from. */}
+        <Subheader>December Campaign</Subheader>
+        <Description>{campaign.description}</Description>
+        <DonationContainer>
+          {/* TODO: Add donation data UI. */}
+          <DonationContent>_DONATION_DATA_</DonationContent>
+          {campaign.active && (
+            // TODO: Open payment modal.
+            <Button className="button--filled" onClick={undefined}>
+              {t('gamHome.listItem.giftButton')}
```

Comment:
fyi Figma designs have the button as plural Gift Meals

Reply:
oh fair, updated this to be plural for the eng translation. would anyone happen to know if we need new copy for chinese?